src/components/_sidebar/YourPortofolio.tsx

```diff
@@ -1,115 +1,207 @@
-              h="24px"
-              w="24px"
-            />
-            <Text
-              fontWeight="semibold"
-              fontSize="16px"
-              color="neutral.400"
-            >
-              {data?.totalSommRewards.value === 0
-                ? "0.00"
-                : data?.totalSommRewards.formatted}{" "}
-              SOMM
+          <Box
+            py="19px"
+            px={8}
+            w="100%"
+            borderTop="1px solid"
+            borderBottom="1px solid"
+            borderColor="surface.secondary"
+          >
+            <Text fontSize={12} fontWeight={600} color="neutral.400">
+              Strategy
             </Text>
-          </HStack>
-        </TransparentSkeleton>
-      </VStack>
-      {Number(data?.totalNetValue?.value) <= 0 && (
-        <CardBase w="full">
-          <Stack>
-            <Text fontWeight="bold" fontSize="16px">
-              Select a strategy to start earning
+          </Box>
+          <LighterSkeleton
+            isLoaded={!isLoading}
+            h={isLoading ? "300px" : "auto"}
+            m="auto"
+            w="full"
+          >
+            <Box w="100%">
+              {data ? (
+                data.strategies.map((strategy) => (
+                  <PortofolioItem
+                    symbol={
+                      strategy.userStrategyData.userData.symbol || ""
+                    }
+                    bondedToken={Number(
+                      strategy?.userStakes?.totalBondedAmount.value ??
+                        0
+                    )}
+                    key={strategy.userStrategyData.strategyData?.name}
+                    icon={
+                      strategy.userStrategyData.strategyData?.logo ??
+                      ""
+                    }
+                    title={
+                      strategy.userStrategyData.strategyData?.name ??
+                      ""
+                    }
+                    netValueUsd={
+                      strategy.userStrategyData.userData
+                        ?.valueWithoutRewards.formatted ?? ""
+                    }
+                    netValueInAsset={
+                      strategy.userStrategyData.userData
+                        ?.netValueWithoutRewardsInAsset.value ?? 0
+                    }
+                    tokenPrice={valueAndFormatted({
+                      value:
+                        strategy.userStrategyData.strategyData?.token
+                          ?.value,
+                      formatted:
+                        strategy.userStrategyData.strategyData?.token
+                          ?.formatted,
+                    })}
+                    slug={
+                      strategy.userStrategyData.strategyData?.slug ??
+                      ""
+                    }
+                    description={
+                      strategy.userStrategyData.strategyData
+                        ?.description ?? ""
+                    }
```

Comment:
not blocking the merge, please improve `PortofolioItem` to just accept `strategy.userStrategyData` and destructure the object inside `PortofolioItem`